### PR TITLE
add new agent config

### PR DESCRIPTION
**Pull request by XiXiaPdx**
### Give us some context

Calls out an existing configuration that could help customers

### CODE REVIEW

**Comment by paperclypse** [comment]
@XiXiaPdx I'm happy to review your PR when it's ready.

File: src/content/docs/agents/java-agent/configuration/java-agent-configuration-config-file.mdx [hunk 1]
```diff
@@ -2626,6 +2626,46 @@ These options are set in the `error_collector` stanza and unless noted otherwise
...
+          <td>
+            false
+          </td>
+        </tr>
+      </tbody>
+    </table>
+
+    With default ignoreErrorPriority as false, the agent reports the last error noticed if noticeError() is called multiple times in a transaction. If set to true, the agent reports the first error noticed if noticeError() is called multiple times in a transaction.
+
+    This setting is dynamic, so running agents will notice changes to `newrelic.yml` without a JVM restart. <--- need to verify for this one
```
Comment (paperclypse) [inline src/content/docs/agents/java-agent/configuration/java-agent-configuration-config-file.mdx:2659]: It looks like the `<---` is breaking our build check @XiXiaPdx. You'll definitely want to remove that once you're ready for this to go out.

**Comment by paperclypse** [comment]
I've made this a draft PR so that others on my team will be aware of its status. When you're done, make it a regular PR and we'll begin the process of reviewing this and merging it in.

**Comment by bradleycamacho** [comment]
@paperclypse @barbnewrelic I wasn't able to review this last week. This is ready to merge after a review.

**Review by paperclypse — APPROVED** [review]
Looks good

**Pull request merged**